hotel_reservation/models/hotel_folio.py

```diff
@@ -9,7 +9,7 @@ class HotelFolio(models.Model):
     _order = "reservation_id desc"
 
     reservation_id = fields.Many2one(
-        "hotel.reservation", "Reservation", ondelete="restrict"
+        "hotel.reservation", "Reservation", ondelete="set null"
```

Comment:
Should'nt it be `cascade` ? If the folio is deleted, the linked reservation should be removed ? I often get confused with those.

Reply:
Right, `set null` seems to be the most sensible.